### PR TITLE
Update package

### PR DESCRIPTION
**Pull request by maniackcrudelis**
## Problem
- *Packaging rules have changed*

## Solution
- *Update to last packaging standard*

## PR Status
- [x] Code finished.
- [x] Tested with Package_check.
- [x] Fix or enhancement tested.
- [x] Upgrade from last version tested.
- [x] Can be reviewed and tested.

## Validation
---
*Minor decision*
- **Upgrade previous version** : 
- [x] **Code review** : JimboJoe
- [x] **Approval (LGTM)** : Kay0u
- [x] **Approval (LGTM)** : JimboJoe
- **CI succeeded** : 
[Build Status/badge/icon)](https://ci-apps-dev.yunohost.org/jenkins/job/etherpad_mypads_ynh%20update_package%20(Official)/)  
When the PR is marked as ready to merge, you have to wait for 3 days before really merging it.

### CODE REVIEW

**Comment by arthurlutz** [comment]
Haven't read all the code (and I don't necessary have the skill to understand it all), but most of the small diffs, added logs look good to me LGTM. 

Can I suggest integrating this into the testing branch so a few select users can test it out ?

**Comment by maniackcrudelis** [comment]
> Can I suggest integrating this into the testing branch so a few select users can test it out ?

Oups ! My bad, this PR should point to testing, not master !

File: scripts/change_url [hunk 1]
```diff
@@ -61,7 +71,7 @@ fi
 #=================================================
 
 ynh_clean_setup () {
-# Nettoyage des résidus d'installation non pris en charge par le script remove.
+# Clean installation remaining that are not handle by the remove script.
```
Comment (JimboJoe) [inline scripts/change_url]:
```suggestion
# Clean installation remainings that are not handled by the remove script.
```

File: scripts/install [hunk 2]
```diff
@@ -16,14 +16,16 @@ source _variables
 #=================================================
 
 ynh_clean_setup () {
-# Nettoyage des résidus d'installation non pris en charge par le script remove.
+# Clean installation remaining that are not handle by the remove script.
```
Comment (JimboJoe) [inline scripts/install]:
```suggestion
# Clean installation remainings that are not handled by the remove script.
```

File: scripts/restore [hunk 3]
```diff
@@ -21,7 +16,7 @@ source ../settings/scripts/_variables
 #=================================================
 
 ynh_clean_setup () {
-# Nettoyage des résidus d'installation non pris en charge par le script remove.
+# Clean installation remaining that are not handle by the remove script.
```
Comment (JimboJoe) [inline scripts/restore]:
```suggestion
# Clean installation remainings that are not handled by the remove script.
```

File: scripts/upgrade [hunk 4]
```diff
@@ -34,71 +35,81 @@ useldap=$(ynh_app_setting_get $app useldap)
...
 #=================================================
 # BACKUP BEFORE UPGRADE THEN ACTIVE TRAP
 #=================================================
+ynh_script_progression --message="Backup the app before upgrading" --weight=35
 
 # Backup the current version of the app
 ynh_backup_before_upgrade
 ynh_clean_setup () {
-	# Cleanup installatioNettoyage des résidus d'installation non pris en charge par le script remove.
+# Clean installation remaining that are not handle by the remove script.
```
Comment (JimboJoe) [inline scripts/upgrade]:
```suggestion
# Clean installation remainings that are not handled by the remove script.
```

**Comment by JimboJoe** [comment]
Will be merged in 3 days.

**Pull request merged**